colors/nord.vim

```diff
@@ -797,3 +797,56 @@ function! NordPalette() abort
   let ret["nord3_bright"] = s:nord3_gui_bright
   return ret
 endfunction
+
+" Treesitter
+if has("nvim") && exists("g:nord_enable_treesitter")
+    !hi link TSError Error
+    !hi link TSPunctDelimiter  Delimiter
+    !hi link TSPunctBracket  Delimiter
+    !hi link TSPunctSpecial Delimiter
+    !hi link TSConstant Constant
+    !hi link TSConstBuiltin Constant
+    !hi link TSConstMacro Constant
+    !hi link TSStringRegex SpecialChar
+    !hi link TSString String
+    !hi link TSStringEscape String
+    !hi link TSCharacter Character
+    !hi link TSNumber Number
+    !hi link TSBoolean Boolean
+    !hi link TSFloat Float
+    !hi link TSAnnotation Structure
+    !hi link TSAttribute Attributes
+    !hi link TSNamespace Structure
+    !hi link TSFuncBuiltin Function
+    !hi link TSFunction Function
+    !hi link TSFuncMacro Function
```

Comment:
This looks good since the [plugin links it to the `Marco` group][1], but `Function` should fit the theme style since the [docs say that the group is for “defined functions (calls and definitions)“][2].
But the exclamation mark must be after the `hi` function call.

```suggestion
    hi! link TSFuncMacro Function
```

[1]: https://github.com/nvim-treesitter/nvim-treesitter/blob/054c2a57/plugin/nvim-treesitter.vim#L47
[2]: https://github.com/nvim-treesitter/nvim-treesitter/blob/3413581/doc/nvim-treesitter.txt#L515